src/SFA.DAS.Provider.PR.Web/Models/AddEmployer/MultipleAccountsShutterPageViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SFA.DAS.Provider.PR.Web.Models.AddEmployer;
+
+public class MultipleAccountsShutterPageViewModel
```

Comment:
This can be record type instead of class

Reply:
ok